README.md

```diff
@@ -30,10 +30,6 @@ and likely lack the precision necessary to be worthwhile.
 This test criteria is also disjoint as the value of an individual test can not be both valid 
 and invalid.
 
-### Redundant Computation
-
-There are several opportunities for redundant computation.
-
 #### Font Validation Tools
```

Comment:
this seemed like direct verification to me, please share your thoughts

Reply:
Had to leave this here since VCS thinks that I moved the below header instead of this section

Reply:
Yeah, it seems to still be confused. I can't see what you're referring to. The diff looks identical to the previous and clicking "View changes" yields an error.